client-api/src/main/java/org/sonarsource/sonarlint/core/clientapi/backend/issue/ChangeIssueStatusParams.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * SonarLint Core - Client API
+ * Copyright (C) 2016-2023 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonarsource.sonarlint.core.clientapi.backend.issue;
+
+import org.sonarsource.sonarlint.core.commons.IssueStatus;
+
+public class ChangeIssueStatusParams {
+
+  private final String configurationScopeId;
+  private final String issueKey;
+  private final IssueStatus newStatus;
+  private final boolean isTaintIssue;
+
+  public ChangeIssueStatusParams(String configurationScopeId, String issueKey, IssueStatus newStatus,
+    boolean isTaintIssue) {
+    this.configurationScopeId = configurationScopeId;
+    this.issueKey = issueKey;
+    this.newStatus = newStatus;
+    this.isTaintIssue = isTaintIssue;
+  }
+
+  public String getConfigurationScopeId() {
+    return configurationScopeId;
+  }
+
+  public String getIssueKey() {
+    return issueKey;
+  }
+
+  public IssueStatus getNewStatus() {
```

Comment:
Usually we try to have the `clientapi` module be self-contained and avoid having dependencies on other modules, like here on `commons` for `IssueStatus`. In fact I added it on this other [PR](https://github.com/SonarSource/sonarlint-core/pull/618/files#diff-afe6deea815f8e733913753da00fcea606ffc26a6f2fc0c1ce36df9c38d6deafR22)

Reply:
Addressed in [e3eedf4](https://github.com/SonarSource/sonarlint-core/pull/612/commits/e3eedf4a8d30bebab82f68b85844e20e2314b3f0)